style.css

```diff
@@ -456,21 +493,51 @@
   margin-bottom: 1rem;
 }
 
+.close {
+  position: absolute;
+  top: 42px;
+  left: 5%;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
 @media (max-width: 767px) {
```

Comment:
- [ ] All page elements should have similar spacing, borders, shadows, and paddings to the original design, kindly consider fixing your styling so all the elements are spaced correctly in the pages